.github/workflows/build.yaml

```diff
@@ -109,7 +108,7 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           ELASTIC_URL: ${{ vars.ELASTIC_URL }}
-          ELASTIC_PORT: ${{ vars.ELASTIC_PORT }}
+          ELASTIC_PORT: 9200
```

Comment:
@paigerube14 should we set the ELASTIC_PORT variable to 9200 instead? Thoughts?

Reply:
I was seeing if I could fix the issues with the elastic/prometheus not properly working. See [comment here](https://github.com/krkn-chaos/krkn-lib/pull/148#issuecomment-2669719561)

The tests were working before, I'll take out these changes

Reply:
If I'm not wrong in your kind action latest update you changed [this kind option](https://github.com/redhat-chaos/actions/blob/c66f512c0fd3ffdce7745769fc339c1d99f4620b/kind/action.yml#L21) so now it expects the kind config file in `.kind-config.yml` so you've to rename the file accordingly and I bet will work.

Reply:
ahh I see what you're saying now Tullio! It uses [this file](https://github.com/krkn-chaos/krkn-lib/blob/main/kind-config.yml) and not the one in the actions folder

I'll change it back, do we want to have more than one node? Or we can't with the porting?

Reply:
https://github.com/redhat-chaos/actions/pull/24